webapp/package.json

```diff
@@ -8,11 +8,14 @@
   "scripts": {
     "build": "webpack --mode=production",
     "lint": "eslint --ignore-pattern node_modules --ignore-pattern dist --ext .js --ext .jsx . --quiet",
-    "fix": "eslint --ignore-pattern node_modules --ignore-pattern dist --ext .js --ext .jsx . --quiet --fix"
+    "fix": "eslint --ignore-pattern node_modules --ignore-pattern dist --ext .js --ext .jsx . --quiet --fix",
+    "test": "jest --forceExit --detectOpenHandles --verbose",
+    "test:watch": "jest --watch",
+    "test-ci": "jest --forceExit --detectOpenHandles --maxWorkers=2"
```

Comment:
I'm 1/5 that the template from https://github.com/mattermost/mattermost-plugin-starter-template/blob/master/webapp/package.json should be used, given that you are already touching the file.

Reply:
The template uses additional scripts like `build:watch`, `debug` and `debug:watch`, that are used by the Makefile there, but not in this plugin. Although these features can be very helpful, I'm 4/5 that we should add them in a separate PR, as they aren't strictly related to tests.
Also, I removed `.ts` and `.tsx` options from these scripts, because we don't have TypeScript files in this plugin.